azure-pipelines.yml

```diff
@@ -0,0 +1,85 @@
+trigger:
+  branches:
+    include:
+      - '*'
+
+pr: none
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+resources:
+  repositories:
+    - repository: platform-libraries-input-mapping
+      type: github
+      endpoint: keboola
+      name: keboola/platform-libraries-input-mapping
+
+variables:
+  DOCKER_BUILDKIT: 1
+  COMPOSE_DOCKER_CLI_BUILD: 1
+
+stages:
+  - stage: build
+    displayName: Build
+    dependsOn: []
+    jobs:
+      - job: checkChanges
+        displayName: 'Check changes'
+        steps:
+          - script: |
+              ./bin/ci-find-changes.sh \
+                input-mapping:libs/input-mapping
+            displayName: 'Find changes'
+            name: findChanges
+
+      - job: build
+        displayName: Build Docker Images
+        steps:
+          - script: docker-compose --profile ci build
+            displayName: Build Docker images
+
+          - script: docker image save $(docker image ls --filter 'reference=keboola/*' --format '{{.Repository}}:latest') -o $(Build.ArtifactStagingDirectory)/docker-images.tar
+            displayName: Save images as artifact
+
+          - publish: $(Build.ArtifactStagingDirectory)
+            artifact: docker-images
+            displayName: Publish artifacts
+
+  - stage: tests_input_mapping
+    displayName: Tests - Input Mapping
+    dependsOn: build
+    condition: and(succeeded(), dependencies.build.outputs['checkChanges.findChanges.changedProjects_input-mapping'])
+    jobs:
+      - job: cs
+        displayName: "Code Style"
+        steps:
+          - script: docker-compose run --rm ci-input-mapping composer phpcs
+            displayName: "IM CS"
```

Comment:
Tohle by melo byt soucasti `libs/input-mapping/azure-pipelines.tests.yml`

Reply:
yeah, just wasn't sure about running it in parallel.  should be run first imho

Reply:
You still can control if you put it in the library `azure-pipelines.test.yml`. But it's an intenral library detail which jobs/steps it runs as its tests, so it shouldn't be in the global pipeline config. We can jump on a quick :phone: if you want more details on it.

Reply:
ah, gotcha, yeah that's a good idea 👍